PersonBook/PersonBook/PersonBook/Program.cs

```diff
@@ -1,78 +1,88 @@
-﻿using PersonBook.Entity;
-using PersonBook.Service;
-using System;
-using System.Collections.Generic;
+﻿using System;
 
 namespace PersonBook
 {
+    using Entity;
+    using Service;
+
     class Program
     {
-       //
         static void Main(string[] args)
         {
-            PersonService pservice = new PersonService();
+            var personService = new PersonService();
+            bool flag = true;
```

Comment:
`flag` as a name seems too common. That could be a more specific name. 
`isContinued` or `isInApp` etc..

Reply:
Done